web-ui/src/pages/FeedbackRequestPage.jsx

```diff
@@ -201,19 +244,22 @@ const handleSubmit = () =>{
     }
 
     const urlIsValid = useCallback(() => {
-      switch (activeStep) {
-        case 1:
-          return hasFor();
-        case 2:
-          return hasFor() && hasTemplate();
-        case 3:
-          return hasFor() && hasTemplate() && hasFrom();
-        case 4:
-          return hasFor() && hasTemplate() && hasFrom() && hasSend();
-        default:
-          return false;
+      if(query) {
+        switch (activeStep) {
+          case 1:
+            return hasFor();
+          case 2:
+            return hasFor() && templateIsValid
+          case 3:
+            return hasFor() && templateIsValid && hasFrom();
+          case 4:
+            return hasFor() && templateIsValid && hasFrom() && hasSend();
+          default:
+            return false;
+        }
       }
-    }, [activeStep, hasFor, hasTemplate, hasFrom, hasSend]);
+      return true;
```

Comment:
In this case, couldn't I omit URL parameters (e.g. I don't put in a `for` parameter) and still be allowed to get to step 1? It seems like we should not be able to get to step 1 without having `for` in the URL.